src/config/express.js

```diff
@@ -3,11 +3,12 @@
 const exphbs = require('express-handlebars');
 const express = require('express');
 const expressValidator = require('express-validator');
+const path = require('path');
 
 module.exports = (app, socketConnections) => {
-    app.engine('handlebars', exphbs({ defaultLayout: '../../src/views/layouts/main' }));
+    app.engine('handlebars', exphbs({ defaultLayout: path.join(__dirname,'../views/layouts/main'), extname: '.handlebars' }));
```

Comment:
It is my personal preference that each part of the path to be joined should be a separate argument of the `path.join` call
`path.join(__dirname, '..', 'views', 'layouts', 'main')`
This is as said only a personal preference - feel free to disregard it if you feel like it